RadarSDK/Include/Radar.h

```diff
@@ -1003,6 +1003,20 @@ logConversionWithNotification
  */
 + (void)setLogLevel:(RadarLogLevel)level;
 
+/**
+ Writes to local logs.
+
+ @param message The message to be logged.
+ */
++ (void)writeLocalLog:(NSString *_Nonnull)message;
```

Comment:
Hmmm, do we need this?

Reply:
It may be worthwhile to allow us to log arbitrary logs into the file system. Seems a little "incomplete" to not implement a "general case" for this feature.